_components/sidenav/sidenav.md

```diff
@@ -7,19 +7,11 @@ component:
 lead: "Hierarchical, vertical navigation to place at the side of a page."
 permalink: /components/side-navigation/
 redirect_from:
-- /sidenav/
-- /components/sidenav/
+  - /sidenav/
+  - /components/sidenav/
```

Comment:
**polish:** Remove added whitespace

```suggestion
- /sidenav/
- /components/sidenav/
```

Reply:
@mahoneycm The indent follows existing conventions.